### PR TITLE
Add development page

### LINKED ISSUE #4: Add new content to the SEF handbook
**Is your feature request related to a problem? Please describe.**

The SEF handbook currently has content which was taken from the GitLab handbook, and was altered slightly. 

**Describe the solution you'd like**

The current SEF handbook content should be replaced by the newly created content. This content can be accessed through this google [doc](https://docs.google.com/document/d/1nRm4rHiDOjjMlRVbCQhUkKBoExg-ufBpLmycis7kod4/edit).

### PR DESCRIPTION
**Pull request by miniskynet**
## Purpose

The purpose of this PR is to fix #4 

## Goals

- A new development sub section was created with new content for the SEF handbook
- Page was renamed from 'Development' to 'Engineering Team'
- Line breaks were added for better readability
- Sidebar position was changed for the pages preceding 'Engineering Team' page
- colorMode was removed due to being deprecated
- Node version was changed on both YML files
- The pre existing 'about us' section content of the engineering team was moved to introduction page
- New content for the 'about us' section was added

### Screenshots

The introduction page with new 'About us' section:

image

The engineering team page with new 'About us' section:

image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by miniskynet** [comment]
@anjula-sack I made some new changes and committed. Is there a better place to mention those changes or should I just edit the current description?

**Comment by anjula-sack** [comment]
> @anjula-sack I made some new changes and committed. Is there a better place to mention those changes or should I just edit the current description?

You can update the description. Can you add the latest screenshots as well?

**Comment by miniskynet** [comment]
Yeah sure I'll add a new screenshot, but nothing much has changed in terms of UI. Just the content has been changed around a bit.

File: docs/organisation/engineering-team.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  - Has made the first pull request
+  - Has introduced themselves on SEF Hive
+- Level 1 - Titan Developer
+  - Has made three pull requests
+  - Has level 0 badge
+- Level 2 - Git Gojira
+  - Has made five pull requests
+  - Has level 1 badge
+- Level 3 - SEF Prodigy
+  - Has passed the quiz
```
Comment (Piumal1999) [inline docs/organisation/engineering-team.md]: Shall we add the quiz link (https://forms.gle/sp8XESD6h5HdQJ5M9) here? @anjula-sack

Comment (anjula-sack) [reply docs/organisation/engineering-team.md]: yeah we need to.

Comment (miniskynet) [reply docs/organisation/engineering-team.md]: okay I'll make an amend. Everything else seems to be okay?

**Comment by miniskynet** [comment]
Guys @anjula-sack @Piumal1999 can you please check now?

**Review by anjula-sack — APPROVED** [review]
Great job! @miniskynet Thanks for getting this done.

**Comment by miniskynet** [comment]
> Great job! @miniskynet Thanks for getting this done.

thank you :)

**Pull request merged**